test/serialize-json.test.js

```diff
@@ -35,10 +35,9 @@ describe('serialize-json', () => {
     bang: null,
     borg: [1, 2, 3, 4],
     boom: new Date('2019-08-01T10:16:30.810Z'),
-    alpha: new URL('https://example.com/Foo'),
     beta: { a: 42, b: 23 },
     gamma: new Map([[1, 42], ['asd', 23], [{}, {}]]),
-    delta: new Set([1, 'asd', {}, new URL('https://example.com/bar')]),
```

Comment:
I would keep the tests if supported.

Reply:
Honestly, that would just result in dead code atm. I would have to add logic to specifically deactivate the tests under node 8 and since we're only testing under node 8 this code would be triggered every time…
Or I would have to polyfill URL type; and since we don't use it ATM that would also just add more dead code.

Before I do one of those two things, I'd rather remove the URL support alltogether…but then on the other hand the logic for that is dead simple (just toString)…

Reply:
we run the tests on node8 in CI, but locally on node10. also we deploy into a node10 container, so I guess we should have both covered. I'll adjust the tests if you don't mind,

Reply:
++1